backend/src/polling_station/mod.rs

```diff
@@ -50,6 +51,38 @@ pub async fn polling_station_list(
     })
 }
 
+/// Get a list of all pollingstations for an election
```

Comment:
This should be updated to a comment about the create endpoint

Reply:
Oops! Fixed in [1667f3f](https://github.com/kiesraad/abacus/pull/538/commits/1667f3f369e8c00f870076eb834fb358adc930a1) and also fixed broken doc links (I guess we never ran `cargo doc` before)